src/app/models/internal/alert-type.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Digital Excellence Copyright (C) 2020 Brend Smits
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU Lesser General Public License as published
+ *   by the Free Software Foundation version 3 of the License.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty
+ *   of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *   See the GNU Lesser General Public License for more details.
+ *
+ *   You can find a copy of the GNU Lesser General Public License
+ *   along with this program, in the LICENSE.md file in the root project directory.
+ *   If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+ */
+
+/**
+ * Enum to define the type of alert used by NGX bootstrap.
+ * See documentation for reference: https://valor-software.com/ngx-bootstrap/#/alerts
+ */
+export enum AlertType {
+    'success',
```

Comment:
I don't know if you have a particular reason for this, but several sources (like: https://www.typescriptlang.org/docs/handbook/enums.html) Mention that enum values must start with an uppercase

Reply:
The reason for this is that NGX Bootstrap uses these predefined strings in order to define the type of the alert. If they start with a capital letter bootstrap does not recognize these types and the alert be left unstyled.

Reply:
Thanks for the explanation!